.docker-compose.datagov-theme.yaml

```diff
@@ -25,9 +27,8 @@ services:
           install_standard_ckan_extension_github -r ckan/ckanext-harvest &&\
           install_standard_ckan_extension_github -r datopian/ckanext-logstash &&\
           install_standard_ckan_extension_github -r ckan/ckanext-dcat ckanext-dcat &&\
-          install_standard_ckan_extension_github -r GSA/ckanext-geodatagov &&\
-          install_standard_ckan_extension_github -r akariv/ckanext-datajson -b datagov &&\
-          install_standard_ckan_extension_github -r akariv/USMetadata -e ckanext-usmetadata &&\
+          install_standard_ckan_extension_github -r datopian/ckanext-geodatagov &&\
+          install_standard_ckan_extension_github -r avdata99/ckanext-datajson -b datagov &&\
```

Comment:
Aren't we using the okfn one in the end?

Reply:
No, OFK data.json extension is not updated. [I updated merging different forks](https://github.com/avdata99/ckanext-datajson/blob/datagov/cherry-picking-from-other-forks.md)
GeoDataGov was forked in order to test it and the send PR to GSA

Reply:
So can we move our datajson extension to the datopian organisation?

Reply:
We need to define who will be the maintainer of this ext.
If GSA will maintain this we can delete and OR to GSA repo

Reply:
This is an official OKF extension, unlikely that GSA will be the maintainers of it.

Reply:
@akariv 
Moved as Datopian repo: https://github.com/datopian/ckanext-datajson
Also migrated to CircleCI: https://circleci.com/gh/datopian/ckanext-datajson